armi/nuclearDataIO/__init__.py

```diff
@@ -115,9 +115,13 @@ def getExpectedGAMISOFileName(cycle=None, suffix=None, xsID=None):
     getExpectedCOMPXSFileName
     getExpectedPMATRXFileName
     """
-    return _findExpectedGammaFileName(
-        neutronics.GAMISO, _getGammaKeywords(cycle, suffix, xsID)
-    )
+    if any(i is not None for i in (cycle, suffix, xsID)):
```

Comment:
Actually, the complexity is needed....if `cycle=0` then `any((0, None, None)) = False`. Any/All apparently check for false-y stuff, not `None` explicitly.